platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/GuestClaimsHandlerExt.java

```diff
@@ -108,6 +108,7 @@ public void init() {
             this.profiles = (Map<String, Object>) fromJson(json);
         } catch (IOException e) {
             LOGGER.debug("Could not find profiles.json during installation: ", e);
+            this.profiles = new HashMap<>();
```

Comment:
In the future, initializing to a on-null value will help with any potential NPEs that unfortunately take place throughout the code base